manim/_config/__init__.pyi

```diff
@@ -0,0 +1,14 @@
+from contextlib import _GeneratorContextManager
+from typing import Any, Union
+
+from rich import Console
+
+from .utils import ManimConfig
+
+logger: Any
+console: Console
+error_console: Console
+config: Any
+frame: Any
+
+def tempconfig(temp: Union[ManimConfig, dict]) -> _GeneratorContextManager: ...
```

Comment:
Why can these types not be added to the actual code file?

Reply:
The capitalized colors are generated in runtime so it does not work.

Reply:
This comment is not about `color.pyi`

Reply:
Ohh sorry. Mypy was interacting weirdly and this seemed to be the most direct solution.